### PR TITLE
Update wasm-as-universal-binary-format-part-2-wapm.md

### PR DESCRIPTION
**Pull request by fschutt**

### CODE REVIEW

File: _posts/wasm-as-universal-binary-format-part-2-wapm.md [hunk 1]
```diff
@@ -10,34 +10,32 @@ In the [first series of the articles](https://wasmer.io/posts/wasm-as-universal-
 
 This was great and very well adopted by the community, but we were missing one big part of the picture: how can we use this new feature to improve the life of developers?
 
-*TL;DR: all WASI packages published to WAPM will have a native executable distribution for each chipset and platform… automatically!*
+*TL;DR: Using the wasmer compiler all WASI packages published to WAPM will now be compiled to a native executable for all available platforms, so that you don't need to ship a runtime with your wasm files*
```
Comment (syrusakbary) [inline _posts/wasm-as-universal-binary-format-part-2-wapm.md:13]:
```suggestion
*TL;DR: Using the wasmer compiler all WASI packages published to WAPM will now be compiled to a native executable for all available platforms, so that you don't need to ship a complete Wasm runtime to run your wasm files*
```
